ocs_ci/deployment/deployment.py

```diff
@@ -397,10 +397,11 @@ def deploy_ocs_via_operator(self, image=None):
 
         """
         ui_deployment = config.DEPLOYMENT.get("ui_deployment")
+        ui_deployment_cond = ui_deployment_conditions()
         live_deployment = config.DEPLOYMENT.get("live_deployment")
         arbiter_deployment = config.DEPLOYMENT.get("arbiter_deployment")
 
-        if ui_deployment:
+        if ui_deployment and ui_deployment_cond:
```

Comment:
Not big deal, but I thing that the function `ui_deployment_conditions()` can be used directly in the `if` statement and then the line 400 is not necessary:
```suggestion
        if ui_deployment and ui_deployment_conditions():
```